lms/djangoapps/oauth2_handler/handlers.py

```diff
@@ -3,14 +3,15 @@
 from django.conf import settings
 from django.core.cache import cache
 
-from courseware.access import has_access
-from student.models import anonymous_id_for_user
-from student.models import UserProfile
 from openedx.core.djangoapps.user_api.models import UserPreference
-from lang_pref import LANGUAGE_KEY
 from xmodule.modulestore.django import modulestore
 from xmodule.course_module import CourseDescriptor
 
+from student.models import anonymous_id_for_user
+from student.models import UserProfile
+from lang_pref import LANGUAGE_KEY
+from student.roles import UserBasedRole, GlobalStaff, CourseStaffRole, CourseInstructorRole
```

Comment:
This is the only major change here. The rest is a result of PyCharm's reformatting.